package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cros-dpsl-js",
-  "version": "1.3.8",
+  "version": "1.4.0",
```

Comment:
nit: Add the released version in `Status codes & messages` section of `src/README.md`, e.g. Released in `dpsl` version: 1.4.0.

Reply:
Done